docs/Learning-Environment-Design-Agents.md

```diff
@@ -175,18 +174,38 @@ For angles that can be outside the range [0,360], you can either reduce the
 angle, or, if the number of turns is significant, increase the maximum value
 used in your normalization formula.
 
-### Multiple Visual Observations
-
-Visual observations use rendered textures directly or from one or more
-cameras in a scene. The Policy vectorizes the textures into a 3D Tensor which
-can be fed into a convolutional neural network (CNN). For more information on
-CNNs, see [this guide](http://cs231n.github.io/convolutional-networks/). You
-can use visual observations along side vector observations.
+#### Vector Observation Summary & Best Practices
```

Comment:
Should this be at the end or at the beginning of the section?

Reply:
Each summary and best practices is at the end of the section. I combined them, because in many cases the best practices are just a summary of the above information.